<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RelationsDialog</class>
 <widget class="QDialog" name="RelationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Friends</string>
    </attribute>
    <widget class="QPushButton" name="friendById">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Find by id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="friendId">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="friendsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>381</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="friendName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="friendByName">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>38</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Find by name</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Groups</string>
    </attribute>
    <widget class="QPushButton" name="groupById">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Find by id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="groupId">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="groupsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>381</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="groupByName">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>38</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Find by name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="groupName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
